### PR TITLE
0048 fix to add spacing above delete buttons

### PR DESCRIPTION
**Pull request by realityexpander**
* Fix spacing issue

### CODE REVIEW

File: app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/reminder_screen/ReminderScreen.kt [hunk 1]
```diff
@@ -416,44 +417,45 @@ fun ReminderScreenContent(
...
+            fun ShowAlertDialogActionType.getTitle() = UiText.Res(
+                R.string.confirm_action_dialog_title_phrase,
+                context.getStringSafe(title.asResIdOrNull),
+                context.getString(R.string.agenda_item_type_reminder)
+            )
+            fun ShowAlertDialogActionType.getMessage() = UiText.Res(
+                R.string.confirm_action_dialog_text_phrase,
+                context.getStringSafe(title.asResIdOrNull).lowercase(),
+                context.getString(R.string.agenda_item_type_reminder).lowercase()
+            )
```
Comment (philipplackner) [inline app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/reminder_screen/ReminderScreen.kt:434]: Why are these functions and not simple variables?

File: app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/task_screen/TaskScreen.kt [hunk 2]
```diff
@@ -439,44 +440,82 @@ fun TaskScreenContent(
...
+            fun ShowAlertDialogActionType.getTitle() = UiText.Res(
+                R.string.confirm_action_dialog_title_phrase,
+                context.getStringSafe(title.asResIdOrNull),
+                context.getString(R.string.agenda_item_type_task)
+            )
+            fun ShowAlertDialogActionType.getMessage() = UiText.Res(
+                R.string.confirm_action_dialog_text_phrase,
+                context.getStringSafe(title.asResIdOrNull).lowercase(),
+                context.getString(R.string.agenda_item_type_task).lowercase()
+            )
```
Comment (philipplackner) [inline app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/task_screen/TaskScreen.kt:494]: Same question here

File: app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/reminder_screen/ReminderScreen.kt (hunk 1 above)
Comment (realityexpander) [reply app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/reminder_screen/ReminderScreen.kt:434]: I was using them in 2 places at one point, but now there is no need for it in two places, so I will refactor these as simple vars.

File: app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/task_screen/TaskScreen.kt (hunk 2 above)
Comment (realityexpander) [reply app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/task_screen/TaskScreen.kt:494]: Will refactor

**Pull request merged**